--- libpreludedb-1.2.6/configure.in.orig	2016-02-03 21:48:00.337869380 +0100
+++ libpreludedb-1.2.6/configure.in	2016-02-03 21:48:43.064534253 +0100
@@ -102,10 +102,8 @@
         # headers.
         AC_MSG_RESULT(Darwin: adding -no-cpp-precomp)
         AX_CHECK_COMPILE_FLAG(-no-cpp-precomp, PCFLAGS="$PCFLAGS -no-cpp-precomp")
-        libext=dylib
         ;;
  *)
-        libext=so
         AC_MSG_RESULT(none needed)
         ;;
 esac
