--- libpreludedb-1.0.0/configure.in.orig	2010-03-10 13:45:13.000000000 +0100
+++ libpreludedb-1.0.0/configure.in	2011-08-10 17:41:34.543209766 +0200
@@ -75,10 +75,8 @@
         # headers.
         AC_MSG_RESULT(Darwin: adding -no-cpp-precomp)
         AX_C_CHECK_FLAG(-no-cpp-precomp, , , PCFLAGS="$PCFLAGS -no-cpp-precomp")
-        libext=dylib
         ;;
  *)
-        libext=so
         AC_MSG_RESULT(none needed)
         ;;
 esac
